app/models/binary/DataSetService.scala

```diff
@@ -374,6 +374,7 @@ class DataSetService @Inject()(organizationDAO: OrganizationDAO,
         "isUnreported" -> Json.toJson(isUnreported(dataSet)),
         "isForeign" -> dataStore.isForeign,
         "jobsEnabled" -> jobsEnabled,
+        "tags" -> dataSet.tags
```

Comment:
So far we have not considered additional fields a breaking change. This would increase the number of new versions by a lot.
I agree that we should unify which routes are versioned and what constitutes a new version. I created a separate issue for this at https://github.com/scalableminds/webknossos/issues/5840